ee/tabby-webserver/src/schema/auth.rs

```diff
@@ -384,6 +400,19 @@ pub trait AuthenticationService: Send + Sync {
         code: String,
         client: Arc<GithubClient>,
     ) -> std::result::Result<GithubAuthResponse, GithubAuthError>;
+
+    async fn get_oauth_credential(
+        &self,
+        provider: OAuthProvider,
+    ) -> Result<Option<OAuthCredential>>;
```

Comment:
do you think it make sense to not returns the actual `client_secret` in the read interface? The front end should just display a dummy masked value?

Reply:
I'l suggest just make client-secret Optional in read / update request.